edx_lint/files/commitlint.config.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  extends: ['@commitlint/config-conventional'],
+
+  helpUrl: 'https://open-edx-proposals.readthedocs.io/en/latest/oep-0051-bp-conventional-commits.html',
+
+  rules: {
+    'type-enum':
+      [2, 'always', [
+        'revert', 'feat', 'fix', 'perf', 'docs', 'test', 'build', 'refactor', 'style', 'chore', 'temp',
+      ]],
+
+    // Default rules we want to suppress:
+    'body-leading-blank': [0, "always"],
+    'body-max-line-length': [0, "always"],
+    'footer-max-line-length': [0, "always"],
+    'footer-leading-blank': [0, "always"],
+    'subject-case': [0, "always", []],
+    'subject-full-stop': [0, "never", '.'],
+  },
+
+  ignores: [
+    // Allow GitHub revert messages, like:
+    //    Revert "introduce a bug"
+    //    Revert "introduce a bug" (#1234)
+    message => /^Revert ".*"( \(#\d+\))?/.test(message),
```

Comment:
Non-blocking suggestion: I mentioned this on the webhooks PR, but it may be worth adding an ignore for the `Merge pull request...` message as well to handle the case where people merged PRs into other PRs before landing the whole change upstream.

Reply:
Ah nevermind, I see your response over there.  commitlint has a default that already ignores that one but not revert messages.